src/v7400/object/model.rs

```diff
@@ -57,4 +59,56 @@ impl<'a> ModelHandle<'a> {
                 _ => None,
             })
     }
+
+    /// Gets the root-most model parent of this object in the scene hierarchy.
+    ///
+    /// If this object has no parents, returns this object.
```

Comment:
Basically OK, but:

* Use "Returns" rather than "Gets" for consistency.
* "ancestor" rather than "parent", since the returned object may "parent of parent of ... of parent" but it is usually not called parent.
* "no parent models" rather than "no parents", since the node may have non-Model parent objects.

```suggestion
    /// Returns the root-most model ancestor of this object in the scene hierarchy.
    ///
    /// If this object has no parent models, returns this object.
```